UI.tsv

```diff
@@ -420,17 +420,17 @@ UI_20150317_000419	Number of Fragments
 UI_20150317_000420	{memo X}Parts 1
 UI_20150317_000421	{memo X}Parts 2
 UI_20150317_000422	{memo X}Parts 3
-UI_20150317_000423	Congratulations! Your level has increased.
+UI_20150317_000423	Congratulations! You have leveled up.
 UI_20150317_000424	Item Alarm
 UI_20150317_000425	{@st64}Details{/}
 UI_20150317_000426	How to Play
 UI_20150317_000427	${@st42b}          Move once with each of the keys{/}
-UI_20150317_000428	${@st42b}         Use a basic attack{/}
-UI_20150317_000429	${@st42b}                Jump up{/}
-UI_20150317_000430	{@st42b}     Use two keys together to move diagonally{/}
+UI_20150317_000428	${@st42b}         Perform a basic attack{/}
+UI_20150317_000429	${@st42b} Jump at the current location or while moving.{/}
```

Comment:
Which is why I have inserted placeholder spacebreaks as a temporary measure. I don't see what's the big deal here, if spacing is incorrect (which it most likely incorrect), someone else or I can myself correct the spacing after seeing the UI. It would be a small PR which I absolutely don't see why this comment even started. I even explained down there that this will require contribution from others to repair the spacing.